frontend/sw360-portlet/src/main/webapp/html/components/includes/components/vulnerabilities.jspf

```diff
@@ -91,7 +91,7 @@
                 </core_rt:when>
                 <core_rt:otherwise>
                         '<div title="<sw360:out value='${vulnerabilityVerificationTooltips.get(vulnerability.externalId).get(vulnerability.intReleaseId)}'/>">'+
-                        '<sw360:DisplayEnum value="${vulnerabilityVerifications.get(vulnerability.externalId).get(vulnerability.intReleaseId)}"/>'+
+                        "<sw360:DisplayEnum value="${vulnerabilityVerifications.get(vulnerability.externalId).get(vulnerability.intReleaseId)}"/>"+
```

Comment:
why did you change single quotes to double quotes in so many places? I don't see the reason.

Reply:
To avoid problems if single quotes are used in the DisplayEnum function.
Earlier I generated a span element with single quotes in the class attribute.
As a result problems occurred and I changed the single quotes to double quotes.

Currently the DisplayEnum function uses double quotes for the class value.

Reply:
Why didn't you use single quotes for the class attribute in DisplayEnum? I mean, it's not that important, but that would contain the change to DisplayEnum, instead of spreading it to multiple files, would it not?

Reply:
The correct solution would be to escape the result of `DisplayEnum`?

Reply:
Right

Reply:
In my opinion it doesn't make to escape the DisplayEnum result here.
The enum values itself can not have single or double quotes in their names.
Also the tooltip text is defined in the CSS file and not in javascript.

In case of double quotes in the tooltip text you need to escape them in the CSS file.

Reply:
until someone toggles the qouting in `DisplayEnum` or maybe use both styles of qouting. Escaping is the safe way.